### ISSUE TITLE
Gosh Bundle Support Thread

### ISSUE THREAD

**Issue by Merith-TK** [jonpalmisc/atto#2]
Basically i am opening this issue as a communication line so that i can keep updating the atto editor plugin,

Edit1) (not relevant) https://pastebin.com/RuAsSvKK )

[Subject of this thread](https://github.com/Merith-TK/gosh-atto)

**Comment by jonpalmisc** [comment]
Haven't gotten a chance to look at this yet, and I'm currently working on a huge refactor of the code. The code is going to be changing drastically again so I'll get back to you once everything is merged.

**Comment by Merith-TK** [comment]
Yeah, right as me and a buddy of mine got 0.3.3 to work, you dropped the
0.4.0 update, but 0.4 helped fix a few other issues,
Like a few for loops didn't exit properly leading to full terminal lockup
(have to externally sigkill the process lol)

But it's working now,

One thing I wish github had was a button to merge the master repo into the
fork repo... It would save so much time lol.

**Comment by Merith-TK** [comment]
I have merged the v0.4.0 edits to my repo and just pushed them,

**Comment by jonpalmisc** [comment]
All the refactoring stuff is merged now and I added an in-program help feature as well (WIP). Might want to check out the changes.

**Comment by jonpalmisc** [comment]
If you find that Atto has weird behavior or is lacking a feature, feel free to open another issue by the way. I have a private issue tracker right now but I'm sure there are things I haven't caught.

**Comment by Merith-TK** [comment]
yeah will check changes later when im at my PC at home.

And nah, its just an issue with how gosh handles exiting the program (see my fork at the areas in v0.4.0 where the program breaks/exits)

**Comment by Merith-TK** [comment]
oh gods, i just `git pull` my local copy of your repo, that is just... a complete rewrite, 

If you have discord, think you could email me your discord? they way i can put you in the chat me and the co-dev are in?

**Comment by jonpalmisc** [comment]
I don't have Discord, but you're welcome to email me at the address on my profile for faster communication.

**Comment by jonpalmisc** [comment]
Are you still working on your project? If not I'd like to close this issue.

**Comment by Merith-TK** [comment]
Taking a small hiatus for school, new simester and I wanna pass this year.

**Comment by Merith-TK** [comment]
Will close for now so i can reopen later
